### PR TITLE
177 speed increase by explicit solving heat loss

### PR DESCRIPTION
**Pull request by samvanderzwan**
Removed the internal grid and adjust the heat loss equation by the explicit solution of the ODE.

### CODE REVIEW

File: src/omotes_simulator_core/solver/network/assets/solver_pipe.py [hunk 1]
```diff
@@ -493,79 +444,34 @@ def _calculate_total_heat_transfer_coefficient(
...
+    def _calculate_total_heat_loss(self, tin: float, mass_flow_rate: float) -> float:
+        r"""Calculate the total heat loss of the pipe.
 
-        Solve the following equation for the internal energy at each grid point:
+        The heat loss is calculated with the following formula:
         .. math::
 
-            \dot{m} \left( e_{i-1} - e_{i} \right) = \alpha A \left( T_{i} -
-            T_{\text{ambient}} \right)
+        \dot{m} c_p (T_{in} - T_{out}) (1 - \exp(-\alpha \pi D L
```
Comment (vanmeerkerk) [inline src/omotes_simulator_core/solver/network/assets/solver_pipe.py:453]: Use \frac{}{} to define a fraction, and \left( \right) to create brackets around T_in and T_out.

**Review by vanmeerkerk — CHANGES_REQUESTED** [review]
Kun je de aangegeven docstring aanpassen?

File: src/omotes_simulator_core/solver/network/assets/solver_pipe.py (hunk 1 above)
Comment (samvanderzwan) [reply src/omotes_simulator_core/solver/network/assets/solver_pipe.py]: fixed

**Review by vanmeerkerk — APPROVED** [review]
Fine

**Pull request merged**